iOS_SDK/OneSignalSDK/Source/OSMigrationController.m

```diff
@@ -40,47 +40,12 @@ + (OSOutcomeEventsCache *)outcomeEventsCache;
 @implementation OSMigrationController
 
 - (void)migrate {
-    [self migrateToVersion_02_14_00_AndGreater];
+    [OneSignalCoreImpl migrate];
+    [OSOutcomes migrate];
     let oneSignalInAppMessages = NSClassFromString(ONE_SIGNAL_IN_APP_MESSAGES_CLASS_NAME);
     if (oneSignalInAppMessages != nil && [oneSignalInAppMessages respondsToSelector:@selector(migrate)]) {
         [oneSignalInAppMessages performSelector:@selector(migrate)];
     }
-    [self saveCurrentSDKVersion];
-}
-
-/**
- * Support renaming of decodable classes for cached data
- */
-- (void)migrateToVersion_02_14_00_AndGreater {
-    let influenceVersion = 21400;
-    let uniqueCacheOutcomeVersion = 21403;
-    long sdkVersion = [OneSignalUserDefaults.initShared getSavedIntegerForKey:OSUD_CACHED_SDK_VERSION defaultValue:0];
-    if (sdkVersion < influenceVersion) {
-        [OneSignalLog onesignalLog:ONE_S_LL_DEBUG message:[NSString stringWithFormat:@"Migrating OSIndirectNotification from version: %ld", sdkVersion]];
-
-        [NSKeyedUnarchiver setClass:[OSIndirectInfluence class] forClassName:@"OSIndirectNotification"];
-        NSArray<OSIndirectInfluence *> * indirectInfluenceData = [[OSInfluenceDataRepository sharedInfluenceDataRepository] lastNotificationsReceivedData];
```

Comment:
I don't see this code or a replacement equivalent in this PR. While the version is pretty old there can definitely be some devices that may go from 2.14 to 5.1.x.

Reply:
Oh yeah, see commit https://github.com/OneSignal/OneSignal-iOS-SDK/pull/1541/commits/9a1ec13d1aaae242cbc91ccc85a295dc4ecd9094

This method `migrateToVersion_02_14_00_AndGreater` is the exact same method as `migrateToVersion_02_14_00_AndGreater` in `[OSOutcomes migrate]` ([see code](https://github.com/OneSignal/OneSignal-iOS-SDK/blob/a1eea2b98b79544a6af7b31b2a8cbcac1254e7d4/iOS_SDK/OneSignalSDK/OneSignalOutcomes/OSOutcomes.m#L60)), which is called by the NSE after we did the modularization. So let's just use `[OSOutcomes migrate]`.

Reply:
Ah my bad, you are right.